README.md

```diff
@@ -49,6 +49,37 @@ sudo apt-get install g++ qt5-default qttools5-dev-tools qt5-doc qtcreator libglu
 
 You can install [librepcb-git](https://aur.archlinux.org/packages/librepcb-git/) from the AUR.
 
+#### Installation on Mac OS X 10.11
+
+1. Install Xcode through the app store and start it at least once 
+   (for the license)
+2. Install [homebrew](https://github.com/Homebrew/brew) (**the** package 
+   manager)
+3. Install *qt5*
+
+        brew update && brew install qt5
+
+4. Link make the toolchain available (you can undo it by linking against *qt*
+   again)
+
+        brew unlink qt
+        brew link --force qt5
+
+5. Compile everything
+
+        mkdir build && cd build
+        qmake ../librepcb.pro -r
+        make -j8
+        cd ../
+
+6. Run the tests
```

Comment:
I would also remove this section "6. Run the teste". It's only relevant for contributors and is thus already described in our [contributing guidelines](https://github.com/LibrePCB/LibrePCB/blob/master/CONTRIBUTING.md).